.gitignore

```diff
@@ -33,6 +36,7 @@
 /doc/*/*.pnr
 /doc/*/*.tex
 /doc/*/*.toc
+/doc/*/make_manuals.out
```

Comment:
Are you sure this is generated by the new build system? If so, please explain how/when.